### ISSUE TITLE
splice junction modeling

### ISSUE THREAD

**Issue by MiqG** [google-deepmind/alphagenome_research#22]
### Description

Hi,

I am working on fine-tuning AlphaGenome on new data. During development, I was managing to overfit all modalities except for splice junctions until realizing that all RoPE parameters (scale and offset) are initialized to zeros. See: https://github.com/google-deepmind/alphagenome_research/blob/455009019acd4c816873d648404ffef2d506606c/src/alphagenome_research/model/heads.py#L1097-L1102

Also, looking at the loss function described in the paper for splice junctions `ratios_loss`and `counts_loss` are scaled by 0.2 and 0.04, respectively. While, in the code by 1 and 0.2, respectively. 

Paper pseudocode:
```python
def junctions_loss(
    x: Array[D, A, N_tissues], targets: Array[D, A, N_tissues]
) -> Array[]:
    ratios_loss = (multinomial_cross_entropy(x, targets, axis=0) +
        multinomial_cross_entropy(x, targets, axis=1))
    counts_loss = (
        poisson_loss(x, targets, axis=0) + poisson_loss(x, targets, axis=1))
    return 0.2 * ratios_loss + 0.04 * counts_loss
```

Splice junction loss in the repo:
https://github.com/google-deepmind/alphagenome_research/blob/455009019acd4c816873d648404ffef2d506606c/src/alphagenome_research/model/heads.py#L1229-L1233

Given this, my questions are:
- Did you initialize scale as 0s as well during training? Could a warmup of the learning rate solve this?
- Which weights did you actually use to train AlphaGenome for the splice junction loss function among the two options?

thanks a lot in advance for your help and for your research! Best,

Miquel

### System info (python version, alphagenome_research version, etc.)

Not applicable.

**Comment by vdutor** [comment]
Hi @MiqG, 

Thanks for raising these issues. We've addressed them in the commits linked above. Please see the commit messages for the rationale behind the changes.

I'll close this issue for now - feel free to open a new one if anything else pops up.

**Comment by MiqG** [comment]
Thanks! Does it work on your end when finetuning a new head?
